### PR TITLE
Update functions billing example

### PR DESCRIPTION
**Pull request by markmirch**
Updating Cloud Functions example for Billing to be consistent with Node sample as well as some additional error checking

## Description

Fixes #<ISSUE-NUMBER>

Note: It's a good idea to open an issue first for discussion.

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md)
- [x] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.6` (see [Test Enviroment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Enviroment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] Please **merge** this PR for me once it is approved.

### CODE REVIEW

File: functions/billing/main.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
 
 # [START functions_billing_limit]
 # [START functions_billing_stop]
-PROJECT_ID = os.getenv('GCP_PROJECT')
+PROJECT_ID = os.getenv('PROJECT_ID')
```
Comment (ace-n) [inline functions/billing/main.py:39]: N.B: this value will need to be set explicitly on deployment. (`GCP_PROJECT` used to be auto-populated in older runtimes.)

File: functions/billing/main.py [hunk 2]
```diff
@@ -83,8 +87,17 @@ def stop_billing(data, context):
 
     projects = billing.projects()
 
-    if __is_billing_enabled(PROJECT_NAME, projects):
-        print(__disable_billing_for_project(PROJECT_NAME, projects))
+    try:
+        billing_enabled = __is_billing_enabled(PROJECT_NAME, projects)
+    except:
+        billing_enabled = True
+        print('Unable to determine if billing is enabled on specified project, assuming true')
```
Comment (ace-n) [inline functions/billing/main.py:94]: Nit: awkward phrasing?

**Comment by leahecole** [comment]
@markmirch pretty please fix lint

File: functions/billing/main.py (hunk 2 above)
Comment (markmirch) [reply functions/billing/main.py]: Reworded

File: functions/billing/main.py (hunk 1 above)
Comment (markmirch) [reply functions/billing/main.py]: Yeah, I'm gonna roll back that change and update the documentation to be more clear around it.

**Comment by markmirch** [comment]
Sorry about that, did proper lint!

**Pull request merged**